WcaOnRails/app/models/regional_organization.rb

```diff
@@ -30,4 +30,13 @@ def start_date_must_be_earlier_than_end_date
   def is_pending?
     start_date.nil?
   end
+
+  def serializable_hash(options = nil)
+    {
+      name: name,
+      website: website,
+      country: country,
+      logo: logo.attached? ? Rails.application.routes.url_helpers.rails_representation_path(logo.variant(resize: "500x300").processed, host: EnvVars.ROOT_URL) : '',
```

Comment:
We want a relative URL, so there's no need to pass the host, no?

Reply:
oh, the fucntion requires host to be set, otherwise it wants us to use `only_path`

Reply:
but I think this is fine, because in production `ROOT_URL` is `worldcubeassociation.org` right

Reply:
Interesting, `_path` is essentially `_url` with only path.

But either way, I've just realised it's an API response, so it's better to return an absolute path. So I think the combination we want is `rails_representation_url` and with explicit `host:`. We already do the same here:

https://github.com/thewca/worldcubeassociation.org/blob/fe6cf26b90f2d78dda44b318b765e2096767555c/WcaOnRails/app/models/person.rb#L273

Reply:
:O aha

Reply:
If we find ourselves using this more, we can do [this](https://andycroll.com/ruby/url-helpers-outside-views-controllers/), which looks pretty elegant. Alternatively we could set global `default_url_options` as described [here](https://github.com/jbranchaud/til/blob/master/rails/set-default-url-options-for-entire-application.md), we currently do it for mailers specifically.

Reply:
I actually tried the latter option on my [previous commit](https://github.com/thewca/worldcubeassociation.org/pull/6939/commits/4fd7def5ad7f68c97493aba3e3bf4163a4a17e46), but I got rid of it, afraid that it would affect code elsewhere. Should I bring it back?

Reply:
Sounds good to me, but it should be more complete, as in:

https://github.com/thewca/worldcubeassociation.org/blob/fe6cf26b90f2d78dda44b318b765e2096767555c/WcaOnRails/config/environments/production.rb#L83-L88

(for each environment file respectively)

With that we most likely no longer need to set `config.action_mailer.default_url_options`, best to check an email preview to verify.

Reply:
I implemented the changes, it works, and the mailer previews are showing up fine, except for a few mailers that are already throwing exceptions beforehand `undefined_method 'preferred_locale' for "dummy_link"`. :)

Reply:
Just to clarify, with the previews I meant specifically that the URLs get the expected hostname.

> Are the config code best placed at where they are right now?

I think so!

Reply:
> Just to clarify, with the previews I meant specifically that the URLs get the expected hostname.

does that mean having the mailer preview at `localhost:3000/rails/mailers/...` on dev?

Reply:
The preview is always going to be there. What I'd check is trigger any email that has a link (reset password or competaition registration?) and check if the URL has the expected hostname.

Reply:
👌 I checked "new registration" email that has a link to "login form", which correctly leads to `localhost:3000/users/sign_in`. Looking good